### PR TITLE
Add configuration and scripts

### PR DESCRIPTION
**Pull request by razvand**
Add Make-based setup files.

### CODE REVIEW

File: README.md
Comment (RaduNichita) [inline README.md:48]: I think we need to separate the common commands (the ones with cloning the repos) from the one specific to platform / architectures.

**Review by RaduNichita — CHANGES_REQUESTED** [review]
All good with me besides that one small comment. Thanks @razvand

File: README.md
Comment (razvand) [reply README.md:48]: I see your point. But, currently, this is how we did for all repositories. I would keep it like this, and, if we think it's a good idea, we will update it all over the place.

Comment (RaduNichita) [reply README.md:48]: Ok, fair point. Then let's merge like it is.

**Review by RaduNichita — APPROVED** [review]
All good now. 

Waiting for #13 to be merged first.

File: README.md
Comment (StefanJum) [inline README.md]:
```suggestion
Use the `.config.helloworld-cpp-qemu-aarch64` configuration file together with `make defconfig` to create the configuration file:
```

File: README.md
Comment (StefanJum) [inline README.md]:
```suggestion
UK_DEFCONFIG=$(pwd)/.config.helloworld-cpp-qemu-aarch64 make defconfig
```

File: README.md
Comment (StefanJum) [inline README.md]:
```suggestion
     .config.helloworld-cpp-fc-x86_64  .config.helloworld-cpp-qemu-aarch64  .config.helloworld-cpp-qemu-x86_64  helloworld-cpp-fc-x86_64.json  kraft.yaml  Makefile  Makefile.uk  README.md [...]
```

**Review by StefanJum — CHANGES_REQUESTED** [review]
See the comments.

**Comment by razvand** [comment]
@StefanJum, fixed.

**Review by StefanJum — APPROVED** [review]
All good, I'll add the tag after #13 is merged and this is rebased.

**Review by StefanJum — APPROVED** [review]
Approved-by: Stefan Jumarea <stefanjumarea02@gmail.com>

**Review by RaduNichita — APPROVED** [review]
All good with rebase. Thanks, @razvand! 

Reviewed-by: Radu Nichita <radunichita99@gmail.com>

**Pull request closed without merging**
